tests/ui/delegation/auxiliary/recursive-delegation-aux.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(fn_delegation)]
+#![allow(incomplete_features)]
+
+fn foo() {
```

Comment:
```suggestion
fn foo() {}
```
Nit: weird formatting.